### ISSUE TITLE
Fix a HTML validation error in the graph page

### ISSUE THREAD

**Issue by davep** [davep/blogmore#507]
If I [run the graph page through the HTML validator](https://validator.w3.org/nu/?doc=https%3A%2F%2Fblog.davep.org%2Fgraph%2F) I get the following error:

```
Error: The aria-label attribute must not be specified on any div element unless the element has a role value other than caption, code, deletion, emphasis, generic, insertion, paragraph, presentation, strong, subscript, or superscript.
From line 1, column 7101; to line 1, column 7150
h-toolbar><div aria-label="Graph legend" class=graph-legend><span 
```

I would read this as the `aria-label` isn't necessary here. This should be cleaned up.
